geode-core/src/main/java/org/apache/geode/cache/client/internal/ExecuteRegionFunctionOp.java

```diff
@@ -516,22 +379,23 @@ protected Object processResponse(Message msg) throws Exception {
                   Throwable cause = ex.getCause();
                   DistributedMember memberID =
                       (DistributedMember) ((ArrayList) resultResponse).get(1);
-                  resultCollector.addResult(memberID, cause);
+                  this.resultCollector.addResult(memberID, cause);
                   FunctionStats
-                      .getFunctionStats(functionId, executor.getRegion().getSystem())
+                      .getFunctionStats(this.functionId, this.executor.getRegion().getSystem())
                       .incResultsReceived();
+                  continue;
                 } else if (((FunctionException) result)
                     .getCause() instanceof InternalFunctionInvocationTargetException) {
                   InternalFunctionInvocationTargetException ifite =
                       (InternalFunctionInvocationTargetException) ex.getCause();
-                  failedNodes.addAll(ifite.getFailedNodeSet());
+                  this.failedNodes.addAll(ifite.getFailedNodeSet());
```

Comment:
Please don't revert all the static analyzer cleanup.

Reply:
Sorry about that. I just sent another commit with those changes back.